src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -493,16 +498,24 @@ public void updateAttackingPlayers()
         }
     }
 
+    @Override
+    public ScorePlayerTeam getTeam()
+    {
+        // This getter will create the team if it doesn't exist. Could do something different though in the future.
+        return checkOrCreateTeam();
+    }
+
     /**
      * Check or create the team.
      */
-    private void checkOrCreateTeam()
+    private ScorePlayerTeam checkOrCreateTeam()
     {
         if (this.world.getScoreboard().getTeam(TEAM_COLONY_NAME + id) == null)
```

Comment:
I think teams are not per-world, so we probably should add colony id + colony dimension to avoid future problems

Reply:
I have not changed this, it was "always" this way.
According to https://minecraft.gamepedia.com/Scoreboard#NBT_format, it seems that the Scoreboard, and thereby the teams, are dimension-agnostic. I hope that colony IDs are unique across multiple dimensions as well?

Reply:
colony ID's arent unique for dimensions, need dimension+colony ID. And ye I saw that it was like that before, but while you're on changing it we should probably add the dimension ID to it

Reply:
Fixed in 1123994 and e54bba8.